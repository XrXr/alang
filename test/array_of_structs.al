struct pack {
    a int
    b int
}

main :: proc () {
    var arr [20]pack

    arr[2].a = 100
    print_int(arr[2].a)
}
