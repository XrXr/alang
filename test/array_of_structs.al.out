100
7171
The id of room[1] is:
2
79797979
91919191
565656
726
